build-aux/m4/bitcoin_qt.m4

```diff
@@ -276,7 +258,7 @@ AC_DEFUN([_BITCOIN_QT_CHECK_QT5],[
       #endif
     ]],
     [[
-      #if QT_VERSION < 0x050200 || QT_VERSION_MAJOR < 5
+      #if QT_VERSION < 0x050501 || QT_VERSION_MAJOR < 5
```

Comment:
is the `QT_VERSION_MAJOR` clause still needed? At first glance, it looks like a leftover from when Qt 4 was still supported

Reply:
I was assuming this is a generic check to see if QT >=5.5.1 is present at all.

Reply:
according to https://doc.qt.io/qt-5/qtglobal.html#QT_VERSION if `QT_MAJOR_VERSION` is 4, then `QT_VERSION` should be something like `0x040102`, so it would already fail the first clause. IOW, I don't think there is any constellation where the first clause is false and the second one is true

Reply:
Could ask @TheCharlatan if there was a reason to do this in 28482efefb15dc2c273a52b96d1aceb995db4968

Reply:
@flack I agree the `|| QT_VERSION_MAJOR < 5` bit seems redundant. That maybe have been in order to support both 4.x and 5.x from certain minor versions onward.

Reply:
I was confused about what the actual supported minor version was at the time. Either way, it is definitely redundant with this pr.